### PR TITLE
Update whitepaper and add scraper summary

### LINKED ISSUE #367: Update whitepaper
Jringo and I worked the whitepaper over. This is good enough for now. Please post on the main website. I would also like the bluepaper copied from the Google drive and placed on the website as a formal resource with a proper link.

Thanks. I would like this done today if possible.

[Gridcoin_Whitepaper_-_Final.pdf](https://github.com/gridcoin-community/Gridcoin-Site/files/10843176/Gridcoin_Whitepaper_-_Final.pdf)

### PR DESCRIPTION
**Pull request by barton2526**
Closes #367

### CODE REVIEW

File: _includes/_footer.htm [hunk 1]
```diff
@@ -142,6 +142,9 @@ <h6>Main Guides</h6>
                     <li>
                         <a href="/assets/docs/grc-bluepaper-section-1.pdf">Gridcoin Staking Bluepaper</a>
                     </li>
+                    <li>
+                        <a href="/assets/docs/scraper-summary.pdf">Executive Scraper Summary</a>
```
Comment (RoboticMind) [inline _includes/_footer.htm:146]: Might we also want to link this to the scraper wiki page?

Comment (barton2526) [reply _includes/_footer.htm:146]: This is under the publications header. The wiki article is informal imo

Comment (jamescowens) [reply _includes/_footer.htm:146]: I think it is ok.

**Review by jamescowens — APPROVED** [review]

**Review by RoboticMind — APPROVED** [review]
uACK

**Pull request merged**